### PR TITLE
Update files to match checkstyle (part 1)

### PR DESCRIPTION
**Pull request by Efnilite**
### Description
Updates some of the worst offenders, who are least likely to be affected by other PRs.

- Removed the copyright header, used IntellIJ's formatting shortcut, and split lines that were >120 chars.
- Moved the Parsers and Serializers in JavaClasses to separate inner classes with IntellIJ's "To named inner class" shortcut.  
- Renamed variables and removed finals

Removes around 700 violations, or 10% of total.

---
**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** none

### CODE REVIEW

File: src/main/java/ch/njol/skript/effects/EffFeed.java [hunk 1]
```diff
@@ -37,41 +19,40 @@
...
-        }
-    }
-
-    @Override
-    public String toString(@Nullable Event e, boolean debug) {
-        return "feed " + players.toString(e, debug) + (beefs != null ? " by " + beefs.toString(e, debug) : "");
-    }
-
+	static {
+		Skript.registerEffect(EffFeed.class, "feed [the] %players% [by %-number% [beef[s]]]");
```
Comment (Moderocky) [inline src/main/java/ch/njol/skript/effects/EffFeed.java:23]:
```suggestion
		Skript.registerEffect(EffFeed.class, "feed [the] %players% [by %-number% [beef[s]]]");
```
Beefs 😭

File: src/main/java/ch/njol/skript/entity/RabbitData.java [hunk 2]
```diff
@@ -24,102 +6,103 @@
...
+	}
+
+	@Override
+	protected boolean init(Literal<?>[] exprs, int matchedPattern, ParseResult parseResult) {
+		type = matchedPattern;
+		return true;
+	}
+
+	@SuppressWarnings("null")
+	@Override
```
Comment (cheeezburga) [inline src/main/java/ch/njol/skript/entity/RabbitData.java:36]:
```suggestion
	@Override
	@SuppressWarnings("null")
```

File: src/main/java/ch/njol/skript/expressions/ExprDateAgoLater.java [hunk 3]
```diff
@@ -36,57 +18,58 @@
...
 
-    @SuppressWarnings("null")
-    private Expression<Timespan> timespan;
-    @Nullable
-    private Expression<Date> date;
-    private boolean ago;
+	@SuppressWarnings("null")
+	private Expression<Timespan> timespan;
+	@Nullable
+	private Expression<Date> date;
```
Comment (cheeezburga) [inline src/main/java/ch/njol/skript/expressions/ExprDateAgoLater.java:35]:
```suggestion
	private @Nullable Expression<Date> date;
```

File: src/main/java/ch/njol/skript/expressions/ExprDateAgoLater.java [hunk 4]
```diff
@@ -36,57 +18,58 @@
...
 
-    @Override
-    @Nullable
-    @SuppressWarnings("null")
-    protected Date[] get(Event e) {
-        Timespan timespan = this.timespan.getSingle(e);
+	@Override
+	@Nullable
+	@SuppressWarnings("null")
+	protected Date[] get(Event e) {
```
Comment (cheeezburga) [inline src/main/java/ch/njol/skript/expressions/ExprDateAgoLater.java:50]:
```suggestion
	@Nullable
	@SuppressWarnings("null")
	protected Date[] get(Event e) {
```
```suggestion
	@SuppressWarnings("null")
	protected Date @Nullable [] get(Event e) {
```

**Review by cheeezburga — APPROVED** [review]
There's some `e`s that could be `event`s, and switches that could be updated, but if this is just a styling thing then looks good

File: src/main/java/ch/njol/skript/effects/EffFeed.java [hunk 5]
```diff
@@ -37,41 +19,40 @@
...
+		if (beefs != null) {
+			Number n = beefs.getSingle(e);
+			if (n == null)
+				return;
+			level = n.intValue();
+		}
+		for (Player player : players.getArray(e)) {
+			player.setFoodLevel(player.getFoodLevel() + level);
+		}
+	}
```
Comment (sovdeeth) [inline src/main/java/ch/njol/skript/effects/EffFeed.java:51]:
```suggestion
	protected void execute(Event event) {
		int level = 20;

		if (beefs != null) {
			Number levelNum = beefs.getSingle(event);
			if (levelNum == null)
				return;
			level = levelNum.intValue();
		}
		for (Player player : players.getArray(event)) {
			player.setFoodLevel(player.getFoodLevel() + level);
		}
	}
```

File: src/main/java/ch/njol/skript/effects/EffFeed.java [hunk 6]
```diff
@@ -37,41 +19,40 @@
...
+
+	@Override
+	public String toString(@Nullable Event e, boolean debug) {
+		return "feed " + players.toString(e, debug) + (beefs != null ? " by " + beefs.toString(e, debug) : "");
+	}
```
Comment (sovdeeth) [inline src/main/java/ch/njol/skript/effects/EffFeed.java:56]:
```suggestion
	public String toString(@Nullable Event event, boolean debug) {
		return "feed " + players.toString(event, debug) + (beefs != null ? " by " + beefs.toString(event, debug) : "");
	}
```

File: src/main/java/ch/njol/skript/entity/RabbitData.java [hunk 7]
```diff
@@ -24,102 +6,103 @@
...
-    
-    private static int intFromType(Rabbit.Type type){
-    	int i = 0;
-    	switch(type){
+	protected boolean equals_i(EntityData<?> obj) {
+		if (!(obj instanceof RabbitData))
+			return false;
+		final RabbitData other = (RabbitData) obj;
+		return type == other.type;
+	}
```
Comment (sovdeeth) [inline src/main/java/ch/njol/skript/entity/RabbitData.java:74]:
```suggestion
	protected boolean equals_i(EntityData<?> otherData) {
		if (!(otherData instanceof RabbitData))
			return false;
		RabbitData other = (RabbitData) otherData;
		return type == other.type;
	}
```

File: src/main/java/ch/njol/skript/entity/RabbitData.java [hunk 8]
```diff
@@ -24,102 +6,103 @@
...
+
+	@Override
+	public boolean isSupertypeOf(EntityData<?> e) {
+		return e instanceof RabbitData && (type == 0 || ((RabbitData) e).type == type);
+	}
```
Comment (sovdeeth) [inline src/main/java/ch/njol/skript/entity/RabbitData.java:79]:
```suggestion
	public boolean isSupertypeOf(EntityData<?> data) {
		return data instanceof RabbitData && (type == 0 || ((RabbitData) data).type == type);
	}
```

File: src/main/java/ch/njol/skript/expressions/ExprDateAgoLater.java (hunk 4 above)
Comment (sovdeeth) [reply src/main/java/ch/njol/skript/expressions/ExprDateAgoLater.java:50]: change event param name too

File: src/main/java/ch/njol/skript/expressions/ExprDateAgoLater.java [hunk 9]
```diff
@@ -36,57 +18,58 @@
...
+	}
 
-    @Override
-    public String toString(@Nullable Event e, boolean debug) {
-        return timespan.toString(e, debug) + " " + (ago ? (date != null ? "before " + date.toString(e, debug) : "ago")
+	@Override
+	public String toString(@Nullable Event e, boolean debug) {
+		return timespan.toString(e, debug) + " " +
+			(ago ? (date != null ? "before " + date.toString(e, debug) : "ago")
 			: (date != null ? "after " + date.toString(e, debug) : "later"));
```
Comment (sovdeeth) [inline src/main/java/ch/njol/skript/expressions/ExprDateAgoLater.java:73]:
```suggestion
	public String toString(@Nullable Event event, boolean debug) {
		return timespan.toString(event, debug) + " " +
			(ago ? (date != null ? "before " + date.toString(event, debug) : "ago")
			: (date != null ? "after " + date.toString(event, debug) : "later"));
```

File: src/main/java/ch/njol/skript/hooks/regions/PreciousStonesHook.java [hunk 10]
```diff
@@ -41,143 +16,143 @@
...
+		return super.init();
+	}
+
+	@Override
+	public String getName() {
+		return "PreciousStones";
+	}
+
+	@Override
+	public boolean canBuild_i(final Player p, final Location l) {
```
Comment (sovdeeth) [inline src/main/java/ch/njol/skript/hooks/regions/PreciousStonesHook.java:42]: better names

File: src/main/java/ch/njol/skript/hooks/regions/PreciousStonesHook.java [hunk 11]
```diff
@@ -41,143 +16,143 @@
...
+		return PreciousStones.API().canBreak(p, l) && PreciousStones.API().canPlace(p, l);
+	}
+
+	@Override
+	public Collection<? extends Region> getRegionsAt_i(final Location l) {
```
Comment (sovdeeth) [inline src/main/java/ch/njol/skript/hooks/regions/PreciousStonesHook.java:47]: better param name

File: src/main/java/ch/njol/skript/hooks/regions/PreciousStonesHook.java [hunk 12]
```diff
@@ -41,143 +16,143 @@
...
-import net.sacredlabyrinth.Phaed.PreciousStones.field.Field;
-import net.sacredlabyrinth.Phaed.PreciousStones.field.FieldFlag;
+import java.io.IOException;
+import java.io.NotSerializableException;
+import java.io.StreamCorruptedException;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public class PreciousStonesHook extends RegionsPlugin<PreciousStones> {
```
Comment (sovdeeth) [inline src/main/java/ch/njol/skript/hooks/regions/PreciousStonesHook.java:26]: this whole class just needs better variable names and removal of extra finals

**Review by sovdeeth — CHANGES_REQUESTED** [review]

**Review by sovdeeth — APPROVED** [review]

**Review by Moderocky — APPROVED** [review]

**Review by cheeezburga — APPROVED** [review]

**Pull request closed without merging**